packages/web-components/components/document/src/DocumentCaptureScreens.js

```diff
@@ -9,12 +9,25 @@ import packageJson from '../../../package.json';
 
 const COMPONENTS_VERSION = packageJson.version;
 
+const smartCameraWeb = document.querySelector('smart-camera-web');
```

Comment:
This one is a bit tricky, since we use the document capture screens in smart camera web, I'm not sure if this will be available during initialisation.

Reply:
Since smart camera web will also introduce a <smart-camera-web /> in the DOM, this should probably be fine. But I've added optional chaining in case it's undefined for any reason.